src/Coordination/CoordinationSettings.cpp

```diff
@@ -22,16 +22,19 @@ void CoordinationSettings::loadFromConfig(const String & config_elem, const Poco
     Poco::Util::AbstractConfiguration::Keys config_keys;
     config.keys(config_elem, config_keys);
 
-    try
+    for (const String & key : config_keys)
     {
-        for (const String & key : config_keys)
+        try
+        {
             set(key, config.getString(config_elem + "." + key));
-    }
-    catch (Exception & e)
-    {
-        if (e.code() == ErrorCodes::UNKNOWN_SETTING)
-            e.addMessage("in Coordination settings config");
-        throw;
+        }
+        catch (Exception & e)
+        {
+            if (e.code() == ErrorCodes::UNKNOWN_SETTING)
```

Comment:
I don't like it. Now we can have typos in settings and nobody will notice.

Reply:
aah that's the reason